Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM jakechampion/heroku-nodejs
```

Comment:
This is a fork of Heroku's NodeJS Dockerfile. The change I have made is to update the version of NodeJS from 0.12 to 5.1